website/docs/_Node/node.mdx

```diff
@@ -1,6 +1,174 @@
 ---
 id: run-node
-title: Run Node (Coming soon...)
+title: Run a Node
```

Comment:
To ensure Docusaurus recognizes the folder and includes it in the sidebar, you might need to change the **folder name** from "_Node" to "Node", as Docusaurus will otherwise ignore folders prefixed with an underscore.